src/user/announcements.rst

```diff
@@ -8,20 +8,31 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2023
 ----
 
+2023-07-12: End-of-life for CentOS 6
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As you may be aware, we have delayed the deprecation of our CentOS 6 build
+system the ``linux64`` platform several times. We have now set a formal deprecation
+date to be June 30, 2024. This date matches the
+`end of extended life-cycle support <https://endoflife.software/operating-systems/linux/red-hat-enterprise-linux-rhel>`_
+from RedHat for RHEL 6. After this date, we build packages against
+CentoOS 7 by default for ``linux64``.
```

Comment:
This is the EOL for CentOS 6 & 7. So don't think we can move to CentOS 7 then

Reply:
I'm happy to change the wording here, but I think jumping past cos7 will be hard for the same reasons cos6 was. We will have a bunch of users with legacy systems and we don't formally need to jump ahead, so it makes more sense to stick with cos7.

Reply:
Also this isn't standard EOL. This date is for their extended lifecycle support maintenance. That goes until June 30, 2028 for cos7. See https://access.redhat.com/support/policy/updates/errata#Extended_Life_Cycle_Phase

The June 30, 2024 date is the end of extended lifecycle support for cos6. 

So going to cos7 in 2028 would be consistent.

Reply:
That link is for RedHat. Not CentOS. The CentOS page confirms what the other link stated. Namely CentOS 7 is EOL 30 June 2024

https://wiki.centos.org/About/Product

<hr>

Appreciate that the situation is messy due to the RedHat changes. However we can't control that. All we can control is the safety of our own infrastructure, which gets significantly harder once CentOS 7 goes EOL. Are we planning to maintain CentOS 7 ourselves? What is our plan?

Reply:
Yes I know the link was for redhat. My point is simply that some people who use conda-forge may still be on cos7 even if it is past EOL.

Reply:
We are not making a promise of 4 years of cos 7 support with this announcement. We are simply saying that when we turn off Cos6, we will use cos 7. We can choose to deprecate cos7 a day later if we please.